src/platform/testing/unit/mocha-setup.js

```diff
@@ -30,8 +36,23 @@ export default function setupJSDom() {
 
   // Prevent warnings from displaying
   /* eslint-disable no-console */
-  console.error = () => {};
-  console.warn = () => {};
+  if (process.env.LOG_LEVEL === 'debug') {
+    console.error = (error, reactError) => {
+      if (reactError instanceof Error) {
+        console.log(filterStackTrace(reactError.stack));
+      } else if (error instanceof Response) {
+        console.log(`Error ${error.status}: ${error.url}`);
+      } else if (error instanceof Error) {
+        console.log(filterStackTrace(error.stack));
+      } else if (error.includes('The above error occurred')) {
```

Comment:
This is the React info with the component tree stack. I'm not sure how useful it really is, but it's pretty short normally.